### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by matthewjreynolds**
Added a landing page for ML and updated the ML structure a bit.

### CODE REVIEW

File: docs/Documentation/Machine_Learning/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+Answer yes to proceed, and you should end up with directions for starting your conda environment pt. Note that these instructions place your environment in the projects folder. This is advisable, as opposed to installing conda environments in their default location in your home directory. See the using Anaconda environment on Kestrel instructions on Kestrel (INCLUDE LINK HERE!) for more information.
+
+Activate the pt conda environment and install PyTorch into the active conda environment:
+```
+conda activate /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/pt
+
+conda install pytorch torchvision torchaudio cpuonly -c pytorch
+```
+Answer yes to proceed, and you should be up and running with PyTorch! The [PyTorch](https://pytorch.org/) webpage has great resources for getting started, including resources on [learning the basics](https://pytorch.org/tutorials/beginner/basics/intro.html) and [PyTorch recipies](https://pytorch.org/tutorials/recipes/recipes_index.html).
```
Comment (yandthj) [inline docs/Documentation/Machine_Learning/index.md:27]:
```suggestion
Answer yes to proceed, and you should be up and running with PyTorch! The [PyTorch](https://pytorch.org/) webpage has great resources for getting started, including resources on [learning the basics](https://pytorch.org/tutorials/beginner/basics/intro.html) and [PyTorch recipes](https://pytorch.org/tutorials/recipes/recipes_index.html).
```

File: docs/Documentation/Machine_Learning/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+###Getting started with TensorFlow
+
+Getting started with TensorFlow is similar to process for PyTorch. The first step is to construct anempty conda environment to work in:
```
Comment (yandthj) [inline docs/Documentation/Machine_Learning/index.md:31]:
```suggestion
Getting started with TensorFlow is similar to the process for PyTorch. The first step is to construct an empty conda environment to work in:
```

File: docs/Documentation/Machine_Learning/index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+```
+Subsequently, activate the tf conda environment, ensure you are running the latest version of pip in your environment, and install the CPU only version of TensorFlow using pip:
+```
+conda activate /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/tf
+pip install --upgrade pip
+pip install tensorflow-cpu
+```
+You should now be up and running with a TensorFlow! Similar to PyTorch, the [TensorFlow webpage](https://www.tensorflow.org/learn) has lots of great resources for getting started, including turotials, basic examples, and more! 
+
+Include a section on how to run the job, e.g., with job script examples or commands for an interactive session.
```
Comment (yandthj) [inline docs/Documentation/Machine_Learning/index.md:45]: Can remove template section

```suggestion
```

File: docs/Documentation/Machine_Learning/index.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+| Kestrel | Swift | Vermillion |
+|:-------:|:-----:|:----------:|
+| 0.0.0   | 0.0.0 | 0.0.0      |
+
+## Advanced
+
+The above examples are simple CPU-based computing environments. To build conda environments for GPUs we refer you to the [PyTorch](https://pytorch.org/get-started/locally/) and [TensorFlow](https://www.tensorflow.org/install) install directions. 
+
+For optimized TensorFlow performance, we recommend using a [containerized version of TensorFlow](https://).
```
Comment (yandthj) [inline docs/Documentation/Machine_Learning/index.md:85]: Missing link

File: docs/Documentation/Machine_Learning/index.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+|:-------:|:-----:|:----------:|
+| 0.0.0   | 0.0.0 | 0.0.0      |
+
+## Advanced
+
+The above examples are simple CPU-based computing environments. To build conda environments for GPUs we refer you to the [PyTorch](https://pytorch.org/get-started/locally/) and [TensorFlow](https://www.tensorflow.org/install) install directions. 
+
+For optimized TensorFlow performance, we recommend using a [containerized version of TensorFlow](https://). 
+
+### Building From Source
```
Comment (yandthj) [inline docs/Documentation/Machine_Learning/index.md:87]: Can remove following sections, since they are just template text

File: mkdocs.yml [hunk 6]
```diff
@@ -147,11 +147,10 @@ nav:
...
       - Machine Learning:
-           - Reinforcement Learning:
-             - Documentation/Machine_Learning/Reinforcement_Learning/index.md
-           - TensorBoard:
-             - Documentation/Machine_Learning/TensorBoard/index.md
-           - Documentation/Machine_Learning/tensorflow.md
+           - Documentation/Machine_Learning/index.md 
+           - Reinforcement Learning: Documentation/Machine_Learning/Reinforcement_Learning/index.md
+           - TensorBoard: Documentation/Machine_Learning/TensorBoard/index.md
+           - Containerized TensorFlow: Documentation/Machine_Learning/index.md
```
Comment (yandthj) [inline mkdocs.yml:153]:
```suggestion
           - Containerized TensorFlow: Documentation/Machine_Learning/Containerized_TensorFlow/index.md
```

File: docs/Documentation/Machine_Learning/index.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+*Machine learning refers to a set of techniques and algorithms that enable computers to automatically learn from data and improve their performance on a specific task over time. Types of machine learning methods include, but are not limited to, supervised learning (algorithms trained on labeled datasets), unsupervised learning (algorithms trained on unlabeled datasets), and reinforcement learning (learning by trial and error). The Computational Science Center at NREL conducts research in these types of machine learning, and also supports the use of machine learning software on Kestrel.*
+
+## Getting Started
+
+This section provides basic examples for getting started with two popular machine learning libraries: [PyTorch](https://pytorch.org/) and [TensorFlow](https://www.tensorflow.org/). Both examples use [Anaconda environments](https://www.anaconda.com/), so if you are not familiar with their use please refer to the NREL HPC page on using Anaconda environments on Kestrel (INCLUDE LINK HERE!) and also the Anaconda guide to [managing environments](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html).
```
[the remark this answers is not in the document]
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:9]: Will commit suggested changes about not being specific to Kestrel

File: docs/Documentation/Machine_Learning/index.md (hunk 1 above)
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:28]: Will fix spelling error

File: docs/Documentation/Machine_Learning/index.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+###Getting started with PyTorch
+
+To begin, we will outline basic steps for building a simple CPU-based conda environment for PyTorch. First, load the anaconda module and create a new conda environment:
+```
+module load conda
+
+conda create -p /projects/YOUR_PROJECT/YOUR_USER_NAME_HERE/FOLDER_FOR_CONDA_ENVIRONMENTS/pt python=3.9
+```
+Answer yes to proceed, and you should end up with directions for starting your conda environment pt. Note that these instructions place your environment in the projects folder. This is advisable, as opposed to installing conda environments in their default location in your home directory. See the using Anaconda environment on Kestrel instructions on Kestrel (INCLUDE LINK HERE!) for more information.
```
[the remark this answers is not in the document]
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:20]: Will remove specific mention of Kestrel.

File: docs/Documentation/Machine_Learning/index.md (hunk 3 above)
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:46]: Will remove template section

File: mkdocs.yml (hunk 6 above)
Comment (matthewjreynolds) [reply mkdocs.yml:153]: Will change formatting of mkdocs.yml

File: docs/Documentation/Machine_Learning/index.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+	```
+!!! note
+	This Getting Started section is only scratching the surface of ML libraries and resources that can be used on Kestrel. Tools such as LightGBM, XGBoost, and scikit-learn work well with conda environments, and other tools such as Flux for the Julia Language can be used on Kestrel as well.
+
+Once you have completed your batch file, submit using
+```
+sbatch <your_batch_file_name>.sb
+```
+
+## Supported Versions
```
[the remark this answers is not in the document]
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:75]: Will remove empty version table

File: docs/Documentation/Machine_Learning/index.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
+# Machine Learning
+
+**Documentation:** [ link to documentation](https://nrel.gov)
```
[the remark this answers is not in the document]
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:3]: Line will be removed.

File: docs/Documentation/Machine_Learning/index.md (hunk 2 above)
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:30]: Fixed typo

File: docs/Documentation/Machine_Learning/index.md (hunk 4 above)
Comment (matthewjreynolds) [reply docs/Documentation/Machine_Learning/index.md:77]: Provided path to Containerized_TensorFlow/index.md

**Review by yandthj — APPROVED** [review]

**Pull request merged**